enterprise-suite/tests/smokecommon

```diff
@@ -112,3 +128,48 @@ test_alertmanager_responding() {
   results=$( curl_query $1 "" | fgrep -q '<title>Alertmanager</title>' )
   T $? alertmanager accessible via $1
 }
+
+# Create services-name-ing ingress if not present
+createIngress() {
+  if ! kubectl get ing -n lightbend "$1-ing"  > /dev/null 2>&1 ; then
+    sed -e "s/{{INGRESS_NAME}}/$1-ing/g;s/{{SERVICE_NAME}}/$1/g;s/{{SERVICE_PORT}}/$2/g" \
+            < resources/gke-ingress-test.template.yaml | kubectl apply -n lightbend -f -
+    echo "create $1-ing for service $1 at port $2"
+    busy_wait limit=180 get_ing_ip "$1-ing"
+  fi
+}
+
+# Get ingress IP
+# Return http://[[public-ip]]
+get_ing_ip() {
+  kubectl get ing "$1" -n lightbend -o json | jq -e '.status.loadBalancer.ingress[0].ip' | tr -d '"'| awk '$1 != "null" {print "http://"$1}'
+}
+
+init_gke_ingress() {
```

Comment:
Does this need a different name?  I see it's used in lots more tests than just the ingress ones.